templates/dart/lib/services/service.dart.twig

```diff
@@ -38,9 +38,9 @@ class {{ service.name | caseUcfirst }} extends Service {
         };
 
 {% if method.type == 'location' %}
-        params.keys.forEach((key) {if (params[key] is int || params[key] is double) {
+        for(var key in params.keys) {if (params[key] is int || params[key] is double) {
```

Comment:
Why a for loop when you could use filter/map/reduce? (Map is faster and resource efficient).

Reply:
removed, as I think this was a leftover